packages/pyroscope-models/src/flamebearer.ts

```diff
@@ -16,6 +16,8 @@ export type Flamebearer = {
    */
   levels: number[][];
   numTicks: number;
+  maxSelf: number;
```

Comment:
in ideal conditions we would break the API here since this is now a required field, but since it's still < 1.0.0 I will leave as it is